ops/terraform/services/base/deploy.groovy

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env groovy
+
+// TODO: Consider further generalizing this and hoisting into a new ops/jenkins/global-pipeline-libs/terraform.groovy
+
+/* Deploys base terraservice
+ * @param args a {@link Map} must include `bfdEnv`, `serviceDirectory`
```

Comment:
technically args should contain 'env' and 'directory', right?

Reply:
You got me. Let me correct that. Thanks!